sample-status-updater-domain/src/main/groovy/life/qbic/samplestatusupdater/update/SampleTrackingService.groovy

```diff
@@ -1,9 +1,14 @@
 package life.qbic.samplestatusupdater.update
 
 import life.qbic.datamodel.samples.Location
+import life.qbic.datamodel.samples.Status
+
+import java.time.Instant
 
 interface SampleTrackingService {
 
     def registerFirstSampleLocation(String sampleCode, Location location) throws SampleUpdateException
 
+    def registerSampleStatus(String sampleCode, Status status, Instant timepoint) throws SampleUpdateException
```

Comment:
```suggestion
    def registerSampleWithFirstStatus(String sampleCode, Status status, Instant timepoint) throws SampleUpdateException
```

Reply:
Do we need the status here? Isnt it automatically set by the system --> metadata received?

Reply:
Ah now I see it. Sorry I was confused on how the classes work together. Makes sense then. So nothing to do here :+1: